runtime/acala/src/xcm_config.rs

```diff
@@ -70,8 +71,41 @@ pub type LocationToAccountId = (
 	SiblingParachainConvertsVia<Sibling, AccountId>,
 	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
 	AccountId32Aliases<RelayNetwork, AccountId>,
+	// Convert `AccountKey20` to `AccountId`
+	AccountKey20Aliases<RelayNetwork, AccountId, EvmAddressMapping<Runtime>>,
```

Comment:
but relay shouldn't have account 20?

Reply:
I'm not sure what form it is. It can be added when needed